core/src/apps/ripple/sign_tx.py

```diff
@@ -69,13 +77,14 @@ def set_canonical_flag(msg: RippleSignTx):
     msg.flags |= helpers.FLAG_FULLY_CANONICAL
```

Comment:
please set `optional uint32 flags [default=0];` in protobuf so that we can avoid the above if

Reply:
Done in 329a038